metadata-ingestion/setup.py

```diff
@@ -383,6 +383,7 @@ def get_long_description():
     "console_scripts": ["datahub = datahub.entrypoints:main"],
     "datahub.ingestion.source.plugins": [
         "file = datahub.ingestion.source.file:GenericFileSource",
+        "affirm-artifact = datahub.ingestion.source.affirm_artifact:AffirmArtifactSource",
```

Comment:
sad that it looks like the ordering of the plugins is not well defined?

saw some was alphabetical then some are not 😂

Reply:
yeah their ordering is a mystery. i can move it to the last to differentiate from the built-in ones.